Scan/PS2/scancode_map.kll

```diff
@@ -0,0 +1,176 @@
+S0x45:U"0"; # 0   F0,45
+S0x46:U"9"; # 9   F0,46
+S0x49:U"Period"; # 0   F0,49
+S0x52:U"Quote"; # '   F0,52
+S0x54:U"LBrace"; # [ FO,54
+S0x55:U"Equals"; # = FO,55
+S0x58:U"CapsLock"; # CAPS   F0,58
+S0x59:U"RShift"; # R SHFT   F0,59
+S0x66:U"Backspace"; # BKSP   F0,66
+S0x69:U"P1"; # KP 1   F0,69
+S0x70:U"P0"; # KP 0   F0,70
+S0x71:U"P."; # KP .   F0,71
+S0x72:U"P2"; # KP 2   F0,72
+S0x73:U"P5"; # KP 5   F0,73
+S0x74:U"P6"; # KP 6   F0,74
+S0x75:U"P8"; # KP 8   F0,75
+S0x76:U"Esc"; # ESC   F0,76
+S0x77:U"NumLock"; # NUM   F0,77
+S0x78:U"F11"; # F11   F0,78
+S0x79:U"P+"; # KP +   F0,79
+S0x83:U"F7"; # F7   F0,83
+S0x0A:U"F8"; # F8   F0,0A
+S0x0B:U"F6"; # F6   F0,0B
+S0x0C:U"F4"; # F4   F0,0C
+S0x0D:U"Tab"; # TAB   F0,0D
+S0x0E:U"Backtick"; # `   F0,0E
+S0x1A:U"Z"; # Z   F0,1A
+S0x1B:U"S"; # S   F0,1B
+S0x1C:U"A"; # A   F0,1C
+S0x1D:U"W"; # W   F0,1D
+S0x1E:U"2"; # 2   F0,1E
+S0x2A:U"V"; # V   F0,2A
+S0x2B:U"F"; # F   F0,2B
+S0x2C:U"T"; # T   F0,2C
+S0x2D:U"R"; # R   F0,2D
+S0x2E:U"5"; # 5   F0,2E
+S0x3A:U"M"; # M   F0,3A
+S0x3B:U"J"; # J   F0,3B
+S0x3C:U"U"; # U   F0,3C
+S0x3D:U"7"; # 7   F0,3D
+S0x3E:U"8"; # 8   F0,3E
+S0x4A:U"Slash"; # /   F0,4A
+S0x4B:U"L"; # L   F0,4B
+S0x4C:U"Semicolon"; # ;   F0,4C
+S0x4D:U"P"; # P   F0,4D
+S0x4E:U"-"; # -   F0,4E
+S0x5A:U"Enter"; # ENTER   F0,5A
+S0x5B:U"RBrace"; # ]   F0,5B
+S0x5D:U"Backslash"; # \   F0,5D
+S0x6B:U"P4"; # KP 4   F0,6B
+S0x6C:U"P7"; # KP 7   F0,6C
+S0x7A:U"P3"; # KP 3   F0,7A
+S0x7B:U"P-"; # KP -   F0,7B
+S0x7C:U"P*"; # KP *   F0,7C
+S0x7D:U"P9"; # KP 9   F0,7D
+S0x7E:U"ScrollLock"; # SCROLL   F0,7E
+
+# Place the extended scancodes at 0x80 + second byte.
+
+# S0xE0,11:U"RAlt"; # R ALT	E0,F0,11
+S0x91:U"RAlt"; # R ALT	E0,F0,11
+
+# S0xE0,14:U"RCtrl"; # R CTRL	E0,F0,14
+S0x94:U"RCtrl"; # R CTRL	E0,F0,14
+
+# S0xE0,1F:U"LGui"; # L GUI	E0,F0,1F
+S0x9F:U"LGui"; # L GUI	E0,F0,1F
+
+# S0xE0,27:U"RGui"; # R GUI	E0,F0,27
+S0xa7:U"RGui"; # R GUI	E0,F0,27
+
+# S0xE0,2F:U""; # APPS	E0,F0,2F
+S0x2F:U"App"; # APPS	E0,F0,2F
+
+# S0xE0,4A:U"P/"; # KP /	E0,F0,4A
+S0xCA:U"P/"; # KP /	E0,F0,4A
+
+# S0xE0,5A:U"PEnter"; # KP EN	E0,F0,5A
+S0xDA:U"PEnter"; # KP EN	E0,F0,5A
+
+# S0xE0,69:U"End"; # END	E0,F0,69
+S0xE9:U"End"; # END	E0,F0,69
+
+# S0xE0,6B:U"Left"; # L ARROW	E0,F0,6B
+S0xEB:U"Left"; # L ARROW	E0,F0,6B
+
+# S0xE0,6C:U"Home"; # HOME	E0,F0,6C
+S0xEC:U"Home"; # HOME	E0,F0,6C
+
+# S0xE0,70:U"Insert"; # INSERT	E0,F0,70
+S0xF0:U"Insert"; # INSERT	E0,F0,70
+
+# S0xE0,71:U"Delete"; # DELETE	E0,F0,71
+S0xF1:U"Delete"; # DELETE	E0,F0,71
+
+# S0xE0,72:U"Down"; # D ARROW	E0,F0,72
+S0xF2:U"Down"; # D ARROW	E0,F0,72
+
+# S0xE0,74:U"Right"; # R ARROW	E0,F0,74
+S0xF4:U"Right"; # R ARROW	E0,F0,74
+
+# S0xE0,75:U"Up"; # U ARROW	E0,F0,75
+S0xF5:U"Up"; # U ARROW	E0,F0,75
+
+# S0xE0,7A:U"PageDown"; # PG DN	E0,F0,7A
+S0xFA:U"PageDown"; # PG DN	E0,F0,7A
+
+# S0xE0,7D:U"PageUp"; # PG UP	E0,F0,7D
+S0xFD:U"PageUp"; # PG UP	E0,F0,7D
+
+##  Pause and print screen are problematic, they cause eratic behaviour.
```

Comment:
Ooo, this is a pretty interesting case. This would be an excellent test for my upcoming unit test infrastructure.
Nothing to worry about for this code review.

(This is what I'm talking about https://github.com/kiibohd/controller/blob/ktype/Scan/TestIn/Tests/test.py)